packages/build/src/log/messages/core.js

```diff
@@ -46,13 +46,15 @@ const logLingeringProcesses = function (logs, processList) {
     logs,
     THEME.errorLine(`
 ** WARNING **
-There are some lingering processes even after the build process finished:
+The build completed successfully but the following processes were accidentally
+left running.
 
 ${processList}
 
-Our builds do not kill your processes automatically, so please make sure
-that nothing is running after your build finishes, or it will be marked as
-failed since something is still running.`),
+Those processes have been forcefully terminated. In case this behavior is
+creating a problem for your build, please see
+[this article](link to community post) for details about why this happens and
+how to fix it.`),
```

Comment:
Some suggested edits: 
(I'm making an assumption that "why this happens" means the termination of processes that were left running. Instead of "see," a [vision-based term](https://netlify-handbook.netlify.app/devex/docs-style-guide/?refresh=login#see%2C-view), we can use something like "refer". We also try to avoid "please" [per style guide](https://netlify-handbook.netlify.app/devex/docs-style-guide/?refresh=login#error-messages).)
I'm pretty sure I'm not getting the spacing right in the suggestion, but it should help provide the words. 😄 

```suggestion
These processes have been forcefully terminated. In case this 
creates a problem for your build, refer to [this article](link to community post) for details about why this process termination happens and how to fix it.`),
```

Reply:
Thanks @rstavchansky!

I am wondering whether the term "forcefully terminated" is ok too. I definitely want to avoid "killed" which is such a negative and weird way to describe ending a piece of logic. The word "terminated" is also quite negative compared to alternatives like "ended" or "completed". However, those last two words do not indicate that the end is abnormal. "aborted" would not work here because it would indicate the process still had some unfinished logic, which it most likely has not. Finally, "terminating" is a common technical term for this instance.

The word "forcefully" is also quite negative and I am not liking it so much. However, I would like some way of indicating that the termination was sudden and might have interrupted the process inner logic. "forceful" is also a common technical term for this instance.

I am not liking this expression so much but could not find better alternatives. Any ideas welcome!